plugins/yellow-review/commands/review/sweep.md

```
@@ -0,0 +1,175 @@
+   beginning with `-`). On match, run
+   `gh pr view "$ARGUMENTS" --json number -q .number`. On mismatch, treat
+   as input error and stop.
+4. **If empty**: Detect from current branch:
+   `gh pr view --json number -q .number`
+
+Validate the resolved value is numeric and non-empty. If not, sanitize
+`$ARGUMENTS` for display (strip every byte outside `[A-Za-z0-9#/:._-]` —
+this prevents terminal escape injection from a malformed input) and
+report:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)